### PR TITLE
audacious: improve examples and page description and add example

### PR DESCRIPTION
**Pull request by vitorhcl**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 4.3.1

### CODE REVIEW

File: pages/common/audacious.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
 # audacious
 
-> An open-source audio player.
+> An open-source audio player. Descendant of XMMS.
```
Comment (fazlearefin) [inline pages/common/audacious.md:3]: I would recommend using an alternative word to `Descendant' which is more tech friendly and easily understood.

Comment (vitorhcl) [reply pages/common/audacious.md:3]: Well, originally, it was a fork of Beep Media Player (BMP). BMP was a (GTK 2) fork of (the GTK 1) XMMS and was abandoned in 2005. It changed a lot of XMMS code although it does have some code that reflect its indirect base.

IMO "descendant" is fine, but as an alternative, what could be used, "indirectly based on" maybe?

Comment (fazlearefin) [reply pages/common/audacious.md:3]: "An open-source audio player inspired by XMMS"

Otherwise "indirectly based on" is also fine

**Review by kbdharun — APPROVED** [review]

File: pages/common/audacious.md [hunk 2]
```diff
@@ -1,12 +1,16 @@
 # audacious
 
-> An open-source audio player.
+> An open-source audio player. Indirectly based of XMMS.
```
Comment (fazlearefin) [inline pages/common/audacious.md:3]:
```suggestion
> An open-source audio player. Indirectly based on XMMS.
```

**Pull request merged**